<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="close_buton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>571</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_buton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_vo_text">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>400</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valve opening = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RCP_text">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RCP.level  = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RCP">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TEP_text">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TEP.level  = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TEP">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuAcceleration">
    <property name="title">
     <string>Acceleration</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionx_1"/>
    <addaction name="actionx_2"/>
    <addaction name="actionx_10"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAcceleration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionx_1">
   <property name="text">
    <string>x 1</string>
   </property>
  </action>
  <action name="actionx_2">
   <property name="text">
    <string>x 2</string>
   </property>
  </action>
  <action name="actionx_10">
   <property name="text">
    <string>x 10</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
